frameworks/Go/fasthttp/benchmark_config.json

```diff
@@ -85,9 +169,87 @@
       "webserver": "None",
       "os": "Linux",
       "database_os": "Linux",
-      "display_name": "FastHTTP",
+      "display_name": "fasthttp",
+      "notes": "",
+      "versus": "go"
+    },
+    "quicktemplate": {
+      "fortune_url": "/fortune-quick",
+      "port": 8080,
+      "approach": "Realistic",
+      "classification": "Platform",
+      "database": "Postgres",
+      "framework": "None",
+      "language": "Go",
+      "flavor": "None",
+      "orm": "Raw",
+      "platform": "None",
+      "webserver": "None",
+      "os": "Linux",
+      "database_os": "Linux",
+      "display_name": "fasthttp",
+      "notes": "",
+      "versus": "go"
+    },
+    "prefork-quicktemplate": {
+      "fortune_url": "/fortune-quick",
+      "port": 8080,
+      "approach": "Realistic",
+      "classification": "Platform",
+      "database": "Postgres",
+      "framework": "None",
+      "language": "Go",
+      "flavor": "None",
+      "orm": "Raw",
+      "platform": "None",
+      "webserver": "None",
+      "os": "Linux",
+      "database_os": "Linux",
+      "display_name": "fasthttp",
+      "notes": "",
+      "versus": "go"
+    },
+    "mongo": {
+      "db_url": "/db",
+      "query_url": "/queries?queries=",
+      "fortune_url": "/fortune",
+      "update_url": "/update?queries=",
+      "port": 8080,
+      "approach": "Realistic",
+      "classification": "Platform",
+      "database": "MongoDB",
+      "framework": "None",
+      "language": "Go",
+      "flavor": "None",
+      "orm": "Raw",
+      "platform": "None",
+      "webserver": "None",
+      "os": "Linux",
+      "database_os": "Linux",
+      "display_name": "fasthttp",
+      "notes": "",
+      "versus": "go"
+    },
+    "mongo-prefork": {
+      "db_url": "/db",
+      "query_url": "/queries?queries=",
+      "fortune_url": "/fortune",
+      "update_url": "/update?queries=",
+      "port": 8080,
+      "approach": "Realistic",
+      "classification": "Platform",
+      "database": "MongoDB",
+      "framework": "None",
+      "language": "Go",
+      "flavor": "None",
+      "orm": "Raw",
+      "platform": "None",
+      "webserver": "None",
+      "os": "Linux",
+      "database_os": "Linux",
+      "display_name": "fasthttp",
       "notes": "",
       "versus": "go"
     }
   }]
-}
\ No newline at end of file
+}
```

Comment:
@savsgio Thanks for your work here. Could you please cut this down to 10 test permutations. I know the other framework you just updated had 12, but we're going to be asking that frameworks cut these down and not accepting any past 10 in the future to try and keep the continuous benchmarking to a reasonable amount of time. Feel free to add and remove as you'd like, so long as it stays at a max of 10.

Reply:
Done!